tests/helpers/Participant.ts

```diff
@@ -0,0 +1,238 @@
+     */
+    private config = {
+        analytics: {
+            disabled: true
+        },
+        debug: true,
+        requireDisplayName: false,
+        testing: {
+            testMode: true
+        },
+        disableAEC: true,
+        disableNS: true,
+        disable1On1Mode: true,
+        disableModeratorIndicator: true,
+        enableTalkWhileMuted: false,
+        gatherStats: true,
+        p2p: {
+            enabled: false,
+            useStunTurn: false
+        },
+        pcStatsInterval: 1500,
+        prejoinConfig: {
+            enabled: false
+        },
+        toolbarConfig: {
+            alwaysVisible: true
+        }
+    };
+
+    /**
+     * Creates a participant with given name.
+     * @param name
+     */
+    constructor(name: string) {
+        this._name = name;
+    }
+
+    /**
+     * Returns participant endpoint ID.
+     * @returns {Promise<string>} The endpoint ID.
+     */
+    async getEndpointId() {
+        if (!this._endpointId) {
+            this._endpointId = await this.driver.execute(() => { // eslint-disable-line arrow-body-style
+                return APP.conference.getMyUserId();
+            });
+        }
+
+        return this._endpointId;
+    }
+
+    /**
+     * The driver it uses.
+     */
+    get driver() {
+        return browser.getInstance(this._name);
+    }
+
+    /**
+     * The name.
+     */
+    get name() {
+        return this._name;
+    }
+
+    /**
+     * Joins conference.
+     */
+    async joinConference(context: { roomName: string }, skipInMeetingChecks = false) {
+        this.context = context;
+
+        const url = urlObjectToString({
+            room: context.roomName,
+            configOverwrite: this.config,
+            interfaceConfigOverwrite: {
+                SHOW_CHROME_EXTENSION_BANNER: false
+            },
+            userInfo: {
+                displayName: this._name
+            }
+        });
+
+        await this.driver.setTimeout({ 'pageLoad': 30000 });
+
+        await this.driver.url(url);
+
+        await this.waitForPageToLoad();
+
+        await this.waitToJoinMUC();
+
+        await this.postLoadProcess(skipInMeetingChecks);
+    }
+
+    /**
+     * Loads stuff after the page loads.
+     * @private
+     */
+    private async postLoadProcess(skipInMeetingChecks) {
+        const driver = this.driver;
+
+        const parallel = [];
+
+        parallel.push(driver.execute((name, sessionId) => {
+            APP.UI.dockToolbar(true);
+
+            // disable keyframe animations (.fadeIn and .fadeOut classes)
+            $('<style>.notransition * { '
+                + 'animation-duration: 0s !important; -webkit-animation-duration: 0s !important; transition:none; '
+                + '} </style>').appendTo(document.head);
+
+            $('body').toggleClass('notransition');
+
+            document.title = `${name}`;
+
+            console.log(`lib-jitsi-meet version: ${JitsiMeetJS.version} sessionId: ${sessionId}`);
```

Comment:
shall we log `navigator.userAgent` to quickly cehck the browser version too?

Reply:
actually we don't need the ljm as we already have it in the logs, same and for the user-agent. I will drop the lib-jitsi-meet one, will leave just the session-id as sometimes it is useful to have it.

Reply:
I added that one :-P I thought maybe you also wanted it here for some reason.

Reply:
Nope, this ends up in the console logs from the browser so no need of dups.